modules/nw-aws-switching-clb-with-nlb.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-switching-clb-with-nlb_{context}"]
+= Switching the Ingress Controller from using a Classic Load Balancer to a Network Load Balancer
+
+You can switch the Ingress Controller that is using a Classic Load Balancer (CLB) to one that uses a Network Load Balancer (NLB) on AWS.
+
+Switching between these load balancers will not delete the IngressController object.
+
+[WARNING]
+====
+This procedure might cause the following issues:
+
+* An outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure.
+
+* Leaked load balancer resources due to a change in the annotation of the service.
+====
+
+.Procedure
+
+. Modify the existing Ingress Controller that you would like to switch to using an NLB. The following example assumes that your default Ingress Controller has an `External` scope and no other customizations:
```

Comment:
```suggestion
. Modify the existing Ingress Controller that you want to switch to using an NLB. The example assumes that your default Ingress Controller has an `External` scope and no other customizations:
```
See `want` in SSG for the replacement https://redhat-documentation.github.io/supplementary-style-guide/#_w also removed `following` based on my previous comment.